Assets/Scripts/SpawnSystem.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+/// <summary>
+/// SpawnSystem spawns objects from a weighted list in a ring around the target. The 
+/// </summary>
+public class SpawnSystem 
+    : MonoBehaviour
+{
+    public AppController Timer;
+    public List<WeightedEnemy> HostileObjects;
+    public AnimationCurve WaveSpawnCount;
+    public AnimationCurve TimeBetweenWaves;
+    [Space]
+    public List<Enemy> ActiveEnemies = new List<Enemy>();
+    [Space]
+    public Transform EarthTransform;
+    public float SpawnDistance;
+    public bool Active;
+
+    private float _timeSinceLastSpawn;
+    private Coroutine SpawnerRoutine;
+
+    public void StartSpawning()
+    {
+        _timeSinceLastSpawn = (int) TimeBetweenWaves.Evaluate(Timer.CurrentTime);
+        Active = true;
+    }
+
+    private void Update()
+    {
+        if(!Active)
+            return;
+
+        if (ActiveEnemies.Count > 2)
+            return;
+
+        _timeSinceLastSpawn += Time.deltaTime;
+
+        if (_timeSinceLastSpawn < (int) TimeBetweenWaves.Evaluate(Timer.CurrentTime))
+            return;
+
+        if (SpawnerRoutine != null)
+            StopCoroutine(SpawnerRoutine);
+
+        _timeSinceLastSpawn = 0;
+        SpawnerRoutine = StartCoroutine(SpawnerCoro());
+    }
+
+    private IEnumerator SpawnerCoro()
+    {
+        var spawnedEnemies = 0;
+        var enemyCount = (int) WaveSpawnCount.Evaluate(Timer.CurrentTime);
+
+        for (var i = 0; i < enemyCount; i++)
+        {
+            var pos = GetPosInCircle(EarthTransform.position, Random.Range(SpawnDistance, SpawnDistance * 1.1f),
+                360f / enemyCount, i);
```

Comment:
no space between division.

Reply:
I don't think we've added one about how to new line the policy doc but I think we new line one argument, we should new line all of them (Not sure about excluding or keeping the first one.)

Reply:
Done